contracts/SCBridgeWallet.sol

```diff
@@ -125,7 +130,26 @@ contract SCBridgeWallet is IAccount {
     challengeExpiry = largestTimeLock + CHALLENGE_WAIT;
```

Comment:
Unrelated to this PR but there's a bug here. If there are no HTLCs, `largestTimeLock=0`, meaning the challenge expiry will be way in the past. This means if you challenge with no HTLCs, there's no wait time for someone to challenge with a later state.

We probably want something like `challengeExpiry = max(largestTimeLock, block.timestamp) + CHALLENGE_WAIT`